server/prisma/schema.prisma

```diff
@@ -19,15 +19,18 @@ model User {
 }
 
 model Post {
-  author        Author   @relation(fields: [authorId], references: [id])
-  authorId      String
-  content       String
-  createdAt     DateTime @default(now())
-  featuredImage String
-  id            String   @id @default(cuid())
-  tags          Tag[]
-  title         String
-  updatedAt     DateTime @updatedAt
+  author          Author   @relation(fields: [authorId], references: [id])
```

Comment:
you need to open this branch on your local env and create a prisma migration file in order for the schema changes to be applied on prod. 
let me know if you need more help here